index.bs

```diff
@@ -264,6 +268,9 @@ div.enum-description > table > thead > tr > th {
     of querying by an element, authors should take the latter one as a correct
     result. Example 2 shows a possible scenario.
 
+    If the element which authors pass is not {{HTMLMediaElement}} or
+    {{AudioContext}}, then these methods will throw a {{TypeError}}.
```

Comment:
Do we need to support extensions of these objects too? I guess `HTMLVideoElement` and `HTMLAudioElement` are extensions?

Reply:
For sure, as those two elements are `HTMLMediaElement` as well. If you feel current statement is not clear, then I will explicitly mention that those extension are supported for `getAutoplayPolicy()`.

Reply:
Yeh for the part of the spec where we're referring to the supported constructors and possibly here too I think it would have to be explicit? Or maybe we could say something along the lines of `HTMLElement`, `AudioContext`, or any other function that has either of those in the prototype chain?

Reply:
Sounds good, I will update it later, thanks!